### ISSUE TITLE
'invalid cross-device link' error when /tmp and target directory aren't on the same partition.

### ISSUE THREAD

**Issue by gfl-chris** [TriggerMail/rules_pyz#52]
Encountering the above error when running pip_generate_wrapper. I believe it's due to the fact that my home directory is mounted on a different partition.

I fixed the issue [here](https://github.com/zenreach/rules_pyz/commit/0c62e39230bd750cdf5f45f346bb08b86b496b48) based on the code from [here](https://go-review.googlesource.com/c/go/+/1591/9/src/io/ioutil/ioutil.go#80).

**Comment by joshburkart** [comment]
Cool! Send PR? (Although @evanj hasn't been responsive lately so we might want to fork this into an official project...)

**Comment by therc** [comment]
It should be OK to fork the project: https://github.com/TriggerMail/rules_pyz/issues/54#issuecomment-455278261

**Comment by evanj** [comment]
Yes please fork it somewhere; I would be happy to accept a PR to post a big note in the `README.md`!
